monitoring/poison_message_queue_finder.py

```diff
@@ -62,5 +67,42 @@ def main():
             print(json.dumps(json_to_log))
 
 
+def get_connection_stats():
+
+    response = requests.get(f"http://{Config.RABBITMQ_HOST}:{Config.RABBITMQ_HTTP_PORT}/api/connections/",
+                            auth=HTTPBasicAuth(Config.RABBITMQ_USER, Config.RABBITMQ_PASSWORD))
+
+    all_connections = response.json()
```

Comment:
Should call `raise_for_status` first, I see the existing script doesn't do that either though.

Reply:
either way, we end up with log noise. I suppose it might be better if the Rabbit Monitor was silent if the HTTP API wasn't working?